python/trezorlib/tests/device_tests/test_msg_recoverydevice_shamir.py

```diff
@@ -221,3 +221,140 @@ def test_abort(self):
         # check that the device is wiped
         features = self.client.call_raw(proto.Initialize())
         assert features.initialized is False
+
+    def test_warnings(self):
+        # 128 bits security, 3 of 6
+        mnemonics = [
+            "extra extend academic bishop cricket bundle tofu goat apart victim enlarge program behavior permit course armed jerky faint language modern",
+            "extra extend academic acne away best indicate impact square oasis prospect painting voting guest either argue username racism enemy eclipse",
+            "extra extend academic arcade born dive legal hush gross briefing talent drug much home firefly toxic analysis idea umbrella slice",
+        ]
+        word_count = len(mnemonics[0].split(" "))
+
+        ret = self.client.call_raw(
+            proto.RecoveryDevice(
+                passphrase_protection=False, pin_protection=False, label="label"
+            )
+        )
+
+        # Confirm Recovery
+        assert isinstance(ret, proto.ButtonRequest)
+        self.client.debug.press_yes()
+        ret = self.client.call_raw(proto.ButtonAck())
+
+        # Homescreen - consider aborting process
```

Comment:
If it's here let's remove it from the 3of6 test above.

Reply:
i put this in a separate test